Patches.py

```diff
@@ -1036,7 +1036,7 @@ def set_entrance_updates(entrances):
         # Remove deku sprout and drop player at SFM after forest completion
         rom.write_int16(0xAC9F96, 0x0608)
 
-    if world.settings.spawn_positions:
+    if (world.settings.spawn_positions != 'off'):
```

Comment:
This doesn't need parentheses.
```suggestion
    if world.settings.spawn_positions != 'off':
```

Reply:
committed. Thank you!